src/frontend/gatsby/src/components/SearchPage/SearchPage.jsx

```diff
@@ -37,6 +41,7 @@ const SearchPage = () => {
   return (
     <div>
       <div className={classes.anchor} id="back-to-top-anchor" />
+      <div className={classes.anchorMobile} id="back-to-top-anchor-mobile" />
```

Comment:
Why not put this class name on the previous element vs. adding a second one (e.g., have a div with two classes)?  @PedroFonsecaDEV any reason that wouldn't work?

Reply:
The className just for the different position, I move scroll to the search bar in the mobile version instead of the very top in the desktop version. 
The reason why I put two divs here is that we need two different ids. one div can not have two different ids. I check the banner component. It has two different ids in two different div/grid too.
![Screenshot 2020-11-27 121322](https://user-images.githubusercontent.com/60892464/100472648-a70aab80-30aa-11eb-9529-4bb7b0cfd485.jpg)

Reply:
I can redesign it to get rid of the extra div. We can only use one .querySelector('#back-to-top-anchor') for both mobile and desktop in ScrollAction.
![image](https://user-images.githubusercontent.com/60892464/101291274-8ee70a80-37d5-11eb-9b6d-5fb2e8b6c090.png)
and then add responsive CSS in each component instead of using two querySelectors. It will remove 1 div in Search page and 1 div in Banner.

Do you think is it necessary? Can I make a issue to redesign the querySelector? @humphd